Dockerfile-alpine.template

```
@@ -96,8 +97,8 @@ RUN set -eux; \
 		| sort -u \
 		| awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
 	)"; \
-	apk add --no-network --virtual .redmine-rundeps $runDeps; \
-	apk del --no-network .build-deps
+	apk add --no-cache --no-network --virtual .redmine-rundeps $runDeps; \
+	apk del --no-cache --no-network .build-deps
```

These result in a warning, but it's otherwise harmless (in the `apk add` case, we know everything we're asking for is already installed or else we've got a bug we need to dig deeper into, and in the `apk del` case we're just asking to remove packages that are also already installed, which shouldn't need the package lists).

```suggestion
	apk add --no-network --virtual .redmine-rundeps $runDeps; \
	apk del --no-network .build-deps
```

---

Then you have to explain like here why there is a warning that's harmless.

I think it's better to have no warning at all :)

Also it's a nice symmetry with `apk add`

---

Sure, but given the choice between a wasted download and a harmless warning, I'd choose the harmless warning. :smile: